lib/main_common.dart

```diff
@@ -14,6 +14,7 @@ import 'package:flutter_localizations/flutter_localizations.dart';
 import 'app_routes.dart';
 import 'package:farmsmart_flutter/farmsmart_localizations.dart';
 
```

Comment:
There're some redux imports 😱  we can get rid of them? Or not yet?

Reply:
Die redux!!!!!! die

Reply:
### FILE REMOVED :D